script/deploy/zksync/DeployLIFuelFacet.s.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import { DeployScriptBase } from "./utils/DeployScriptBase.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+import { LIFuelFacet } from "lifi/Facets/LIFuelFacet.sol";
+
+contract DeployScript is DeployScriptBase {
+    using stdJson for string;
+
+    constructor() DeployScriptBase("LIFuelFacet") {}
+
+    function run()
+        public
+        returns (LIFuelFacet deployed, bytes memory constructorArgs)
+    {
+        constructorArgs = abi.encodePacked("");
+        deployed = LIFuelFacet(deploy(type(LIFuelFacet).creationCode));
+    }
+}
```

Comment:
Do we still need LIFuel facet, scripts etc. As far as I know it's deprecated

Reply:
it's not yet deprecated. We are waiting for the BE team to remove it, then we will remove it from our codebase. If we would remove it first then our types package would be updated and the BE would potentially run into issues with our latest types package version.

Reply:
I can remove this deploy script from the zksync side without issues though.